### PR TITLE
Add missing ballerina documentation

### PR DESCRIPTION
**Pull request by Manuri**
## Purpose
$Subject

### CODE REVIEW

File: stdlib/database/sql/src/main/ballerina/sql/actions.bal [hunk 1]
```diff
@@ -22,6 +22,7 @@ public type CallerActions object {
     #
     # + sqlQuery - The SQL stored procedure to execute
     # + recordType - Array of record types of the returned tables if there is any
+    # + parameters - The parameters to be passed to the procedure/function call. The number of parameters is variable
```
Comment (anupama-pathirage) [inline stdlib/database/sql/src/main/ballerina/sql/actions.bal]: "The number of parameters is variable" Is this sentence grammatically correct?

Comment (Manuri) [reply stdlib/database/sql/src/main/ballerina/sql/actions.bal]: Yes, variable is an adjective. (We use it as a noun as well)

File: stdlib/transactions/src/main/ballerina/transactions/commons.bal [hunk 2]
```diff
@@ -164,6 +164,11 @@ function getParticipantProtocolAt(string protocolName, int transactionBlockId) r
 
 # The initiator will create a new transaction context by calling this function. At this point, a transaction object
 # corresponding to the coordinationType will also be created and stored as an initiated transaction.
+#
+# + coordinationType - The type of the coordination relevant to the transaction block for which this TransactionContext
+#                      is being created for
+# + transactionBlockId - The ID of the transaction block
+# + return - The created TransactionContext or error if the coordinationType is invalid
```
Comment (anupama-pathirage) [inline stdlib/transactions/src/main/ballerina/transactions/commons.bal]: Shall we reword this to "TransactionContext if the coordination type is valid or an error in case of an invalid coordination type."

File: stdlib/transactions/src/main/ballerina/transactions/commons.bal [hunk 3]
```diff
@@ -289,11 +299,13 @@ function getParticipant2pcClient(string participantURL) returns Participant2pcCl
     }
 }
 
-# Registers a participant with the initiator's coordinator. This function will be called by the participant
+# Registers a participant with the initiator's coordinator. This function will be called by the participant.
 #
-# + transactionId - - ID of the transaction to which this participant is registering with
-# + transactionBlockId - - The local ID of the transaction block on the participant
-# + registerAtURL - - The URL of the initiator to which this participant will register with
+# + transactionId - ID of the transaction to which this participant is registering with.
```
Comment (anupama-pathirage) [inline stdlib/transactions/src/main/ballerina/transactions/commons.bal]: Shall we change this as " Global transaction ID to which this participant is registering with"

File: stdlib/transactions/src/main/ballerina/transactions/commons.bal [hunk 4]
```diff
@@ -289,11 +299,13 @@ function getParticipant2pcClient(string participantURL) returns Participant2pcCl
...
+# + transactionBlockId - The local ID of the transaction block on the participant.
+# + registerAtURL - The URL of the initiator to which this participant will register with.
```
Comment (anupama-pathirage) [inline stdlib/transactions/src/main/ballerina/transactions/commons.bal]: This should be "The URL of the coordinator"
